src/traits/lowering-module.md

```diff
@@ -9,21 +9,16 @@ created in the [`rustc_traits::lowering`][lowering] module.
 ## The `program_clauses_for` query
 
 The main entry point is the `program_clauses_for` [query], which –
-given a def-id – produces a set of Chalk program clauses. These
-queries are tested using a
-[dedicated unit-testing mechanism, described below](#unit-tests).  The
+given a def-id – produces a set of Chalk program clauses. The
 query is invoked on a `DefId` that identifies something like a trait,
 an impl, or an associated item definition. It then produces and
 returns a vector of program clauses.
 
 [query]: ../query.html
 
-<a name="unit-tests"></a>
-
 ## Unit tests
 
```

Comment:
Maybe add a disclaimer to show Chalk doesn't have unit tests for the moment?

Reply:
Good point! Added a note.